test/services/optout/OptOutPropositionSpec.scala

```diff
@@ -76,6 +76,15 @@ class OptOutPropositionSpec extends UnitSpec {
         case _ => throw new RuntimeException(s"Unexpected ITSA status data $input")
       }
 
+    def parseExpectedOutcome(input: String): String =
```

Comment:
I would just have re-used `parseItsaStatus()` above. In both cases you're just trying to parse an ITSA Status. In face, the above doesn't distinguish between the CY-1, CY and CY+1 cases where it's currently used. I think it's fine to use it for the expected statuses. If there are any errors, then the stack trace should lead to exactly which column caused the issue.

Reply:
changed this now 👍